identity-wallet/src/error.rs

```
@@ -100,6 +100,8 @@ pub enum AppError {
     TrustListNotFoundError(String),
     #[error("Failed to migrate AppState version `{0}` to version `{1}`: {2}")]
     AppStateMigrationError(u32, u32, String),
+    #[error("Provided code is invalid")]
+    InvalidTransactionCode(#[source] anyhow::Error),
```

This error is not used anywhere so let's leave it out of this PR. Once we actually implement proper error handling for this we can add it back in but I suspect we would not want to return an error, but instead prompt the user to try to enter their pincode again (via `CurrentUserPrompt`)